src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final double radius;
+    private String color;
```

Comment:
Since the 'color' property is set in the constructor and not expected to change, it should be declared as final to ensure the immutability of the class instances.